File: ServiceBus/MessagingExpress/Queue/send-message.js
const {ServiceBusClient} = require("@azure/service-bus");
const connectionString = ""

const queueName = "queue001";
const sbClient = new ServiceBusClient(connectionString);
const sender = sbClient.createSender(queueName);

async function sendMessage(){

    // create an array of messages
    const messages = [
        {body: "Hello, this is first message."},
        {body: "Hi, this is second message."}
    ];

    // send messages - individual messages will be created in queue.
    await sender.sendMessages(messages);
}

sendMessage()
    .then(() => {
        return sender.close();
    })
    .then(() => {
        return sbClient.close();
    })
    .catch((err) => {
        console.error(err);
    });

File: ServiceBus/MessagingPython/Queue/send-message.py
from azure.servicebus import ServiceBusClient, ServiceBusMessage

connection_str = ""
queue_name = "queue001"

def send_message(connection_str, queue_name):
    servicebus_client = ServiceBusClient.from_connection_string(connection_str)
    sender = servicebus_client.get_queue_sender(queue_name)
    message = ServiceBusMessage("Hello from Python")
    sender.send_messages(message)
    sender.close()
    servicebus_client.close()

send_message(connection_str, queue_name)
